OPHD/States/MapViewState.cpp

```diff
@@ -412,26 +412,33 @@ void MapViewState::onKeyDown(NAS2D::EventHandler::KeyCode key, NAS2D::EventHandl
 		return;
 	}
 
+	int shiftMoveScalar = 1;
+	if (NAS2D::EventHandler::shift(mod))
+	{
+		shiftMoveScalar = 5;
+	}
```

Comment:
This can be shortened with a ternary operator, which also allows the value to be marked `const`:
```cpp
const int shiftMoveScalar = NAS2D::EventHandler::shift(mod) ? 5 : 1;
```